src/karts/controller/skidding_ai.cpp

```diff
@@ -1172,80 +1192,252 @@ void SkiddingAI::handleItems(const float dt)
         }
         return;
     }
-
-    // Tactic 1: wait ten seconds, then use item
+   
+   int ai_skill = 0;
+   if (m_ai_properties->m_item_usage_skill > 0)
+   {
+      if (m_ai_properties->m_item_usage_skill > 5)
+      {
+         ai_skill = 5;
+      }
+      else
+      {
+         ai_skill = m_ai_properties->m_item_usage_skill;
+      }
+   }
+   
+   if (m_kart->getBoostAI() == true)
```

Comment:
Very minor, could be written more concisely:

if (m_kart->getBoostAI() && ai_skill < 5) ai_skill++;